main.go

```diff
@@ -176,6 +153,9 @@ func main() {
 		os.Exit(1)
 	}
 
+	cfg.Burst = int(opts.clientBurst)
+	cfg.QPS = opts.clientQPS
```

Comment:
Where cfg.QPS  used?

Reply:
`cfg` gets assigned to `targetK8sConfig` which then gets used to create Kubernetes API client instances for `ConfigurationPolicyReconciler`.